README.md

```diff
@@ -1,5 +1,18 @@
 # Wortels
-This is the design repo for wortels, a specification for some random project.
+
+This is the design repo for Wortels - a novel design for a stateless WebAssembly-based consensus computer.
+
+It is intended to build on learnings from Ethereum, an existing blockchain computer.
+
+The goals are:
+- Can operate as a pegged chain or sidechain
+- Stateless chain
+- No native token (aka "account abstraction")
+- WebAssembly as an execution engine (this should focus in linking code)
+- Compatibility with Ethereum's EVM is not a primary goal
```

Comment:
Maybe not include `this is not a goal` as a goal, alternatively note it elsewhere in the doc.